BaseClasses.py

```diff
@@ -829,7 +808,7 @@ def is_not_bunny(self, region: Region, player: int) -> bool:
         if self.has('Moon Pearl', player):
             return True
 
-        return region.is_light_world if self.world.mode[player] != 'inverted' else region.is_dark_world
+        return region.is_light_world if self.world.world_state[player] != 2 else region.is_dark_world
```

Comment:
The usage of `WorldState.inverted` is inconsistent here and in many other places. They should all be changed to use the property rather than direct comparison.

Reply:
i didn't think to create the property until it was too late ok? i forgot some...